episodes/lesson-design.md

```diff
@@ -22,79 +22,58 @@ After completing this episode, participants should be able to...
+
+This training promotes an iterative backward design process where, after identifying the target audience of our lesson, we
 
-The last two phases of Nicholls' paradigm involve
-assessing learner progress towards the desired learning outcomes and
-evaluating the stated objectives and current content in light of the results of that assessment.
-In The Carpentries, most workshops are relatively short-format,
-without room for an extensive assessment after the teaching has finished
-(a _summative_ assessment).
-To account for this, our lessons place an emphasis on _formative_ assessment:
-assessment of learner progress that takes place _while the teaching is still going on_,
-to give instructors opportunities to evaluate the teaching and lesson content
-before the end of the workshop.
-
-To account for this,
-we have adapted Nicholls' five phases in this training,
-to place an emphasis on assessing learning during a workshop:
-
-1. Define desired learning outcomes
-2. Design assessments to determine progress towards desired outcomes
-3. Write content to lead learners from one of these assessments to the next
-4. Assess learner progress towards outcomes during teaching
-5. (After the break) evaluate how closely the outcomes meet the objectives
-
-![An overview of the iterative process of lesson design and development 
-, adapted from Nicholls' five phases,
-that will be presented in this training.
+1. Define desired learning outcomes.
+2. Design assessments to determine progress towards desired outcomes.
+3. Write content to lead learners from one of these assessments to the next.
+4. Assess learner progress towards outcomes during teaching.
+5. (After the break) evaluate how closely the outcomes meet the objectives.
+
+![An overview of the iterative process of lesson design and development, adapted from Nicholls' five phases, that will be presented in this training.
 ](fig/cldt-design-process.svg){
 alt="A flow diagram presenting the process of lesson design and development used in this training."
 width="67%"
 }
 
-Note the cyclical nature of this process:
-you will complete one iteration through this cycle during this training
-(though probably for a limited part of your lesson, rather than the whole thing).
-Note also that teaching the content is an essential intermediate step in the process:
-the importance of feedback gathered while teaching the lesson
-will be a common refrain throughout this training.
+The Carpentries community specialises in relatively short-format workshops, typically lasting less than a week and including no opportunities for any assessment after the teaching has finished (a _summative_ assessment).
+To account for this, our lessons (and this training) place an emphasis on _formative_ assessment: assessment of learner progress that takes place _while the teaching is still going on_, to give instructors opportunities to evaluate the teaching and lesson content before the end of the workshop.
```

Comment:
This seems out of place with the flow. Do we need it here? If so, maybe need to tie to the material above and below, which seem to be related already